src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -728,14 +729,14 @@ private boolean isSeparatorBeforeImport(int line) {
      * @param previous previous non-static but current is static (above), or
      *     previous static but current is non-static (under).
      * @param name the name of the current import.
-     * @param line the line of the current import.
+     * @param ast the line of the current import.
```

Comment:
Same.

Reply:
done. removed this line

Reply:
This line can't be removed. We have to document all method parameters.

Reply:
Ok I will document this, and will push the changes